### DISCUSSION TITLE
article/notionnext-twikoo

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#50] [Announcements]
# article/notionnext-twikoo

NotionNext现在支持 Twikoo啦，很好用的评论 插件

https://tangly1024.com/article/notionnext-twikoo

**Comment by dreamer2q** [comment]
测试～

**Comment by dreamer2q** [comment]
看起来暗黑模式的适配不太完善～

**Comment by wzf19901221666** [comment]
如何映射二级域名，那边设置好后 我这边没有对应的复制按钮

**Reply by itpPhange** [reply]
映射二级域名，可以参考博主的域名映射来就是了

**Comment by wzf19901221666** [comment]
comment faile

**Comment by wzf19901221666** [comment]
这边仓库好像也没有用别人的，是不是因为域名没有映射 ，直接用 twikoo.我的网站，所以comment fail：0 的错误

**Reply by itpPhange** [reply]
我一开始没有映射二级域名也是报这个错误，然后我去映射了就可以实现评论了。

**Comment by lee-forever** [comment]
很棒的教程。如果出现问题推荐搭配【Twikoo Vercel 部署教程】 https://www.bilibili.com/video/BV1Fh411e7ZH 食用

**Comment by lee-forever** [comment]
同样是个帮助文档https://twikoo.js.org/

**Comment by XinzheGao** [comment]
测试～

**Comment by honest4802** [comment]
test

**Comment by itpPhange** [comment]
:apple 测试

**Comment by cloud-oc** [comment]
Twikoo好像失效了，输入框不显示了
